NodeFirefoxDebug/entry_point.sh

```diff
@@ -3,6 +3,7 @@
 # IMPORTANT: Change this file only in directory NodeDebug!
 
 source /opt/bin/functions.sh
+/opt/bin/generate_config > /opt/selenium/config.json
```

Comment:
Would these lines replace the `RUN /opt/bin/generate_config > /opt/selenium/config.json` lines? or would they be supplemental